sys/net/gnrc/network_layer/ipv6/ext/frag/Kconfig

```
@@ -0,0 +1,43 @@
+# Copyright (c) 2019 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_IPV6_EXT_FRAG
+    bool "Configure GNRC IPv6 fragmentation and reassembly"
+    depends on MODULE_GNRC_IPV6_EXT_FRAG
+    help
+        Configure GNRC IPv6 fragmentation and reassembly via Kconfig.
+
+if KCONFIG_MODULE_GNRC_IPV6_EXT_FRAG
+
+config GNRC_IPV6_EXT_FRAG_SEND_SIZE
+    int "IPv6 fragmentation send buffer size"
```

Without the context of code and given numbers this can be misleading I just noticed when opening `make menuconfig`
```suggestion
    int "Number of entries IPv6 in the send buffer"
```

---

Changed